Source/Core/Core/HW/Memmap.cpp

```diff
@@ -102,17 +103,60 @@ bool IsInitialized()
 }
 
 
-// We don't declare the IO region in here since its handled by other means.
+// Dolphin allocates memory to represent four regions:
+// - 24MB RAM, available on Gamecube and Wii
+// - 64MB "EXRAM", RAM only available on Wii
+// - 64MB FakeVMem, allocated when MMU support is turned off. This is used
```

Comment:
Why is this 64MB? The "common library" that it refers to is the GC VM SDK, which supports less than 16MB (the physical size of ARAM used for backing) of virtual memory.